docs/source/api_ref.rst

```diff
@@ -89,3 +89,9 @@ Utils API
 
 .. automodule:: ivadomed.utils
 
+
+Maths API
+++++++++++++++++++++
```

Comment:
i think the convention is to use the same number of characters as the text
```suggestion
++++++++++++++++++++
```

Reply:
done in: 503aaa8